code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -74,11 +74,11 @@
 		agent = preset_agent
 
 
-	scientist.assigned_role = "MODE"
+	scientist.assigned_role = "Abductor Scientist"
```

Comment:
I tried this on my test server with event-type Abductors rather than Round-type abductors, and nothing seemed to break.

How would you recommend fixing this, because at the moment the round end screen says "Fox McCloud was Fox McCloud the MODE."